Sources/StreamChat/Utils/Codable+Extensions.swift

```diff
@@ -4,12 +4,26 @@
 
 import Foundation
 
+final class StreamJSONDecoder: JSONDecoder {
+    override func decode<T>(_ type: T.Type, from data: Data) throws -> T where T: Decodable {
```

Comment:
why did you have this here and not in the response decoder?

Reply:
Can you clarify?